src/obfl-specification.html

```diff
@@ -1195,6 +1195,36 @@ <h4 id="L722" class="include">evaluate</h4>
           <dd class="markup note">This attribute is only available in the context of a field element.</dd>
       </dl>
     </dd>
+  <dt>Variables</dt>
+    <dd class="markup note">These are commonly available variables. Depending
+      on the context, other variables may be available as well.</dd>
+    <dd class="markup note">For readability, the examples below have plain text
+      instead of braille.</dd>
+    <dd><dl>
+        <dt>page</dt>
+          <dd>The variable holding the current page number. It can be
+            used in the expression, for instance, <code>This is page
```

Comment:
Instead of repeating the "It can be used in the expression" in each of these examples why not put it directly beneath "Variables"? Something like:

> These are the available variables for use in the OBFL Evaluation Language expression.